content/guides/content/adding-papers.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Adding a new paper"
+author: "Jonah Paper"
+description: "Instructions on how to create a new paper to AnVIL"
+---
+
+# Adding a new paper
+If you wrote a paper based on analysis from AnVIL or that has some impact for AnVIL, you can add it to the [Publications](/overview/publications) page.
+
+### Overview
+All publications are stored in a single markdown file, and appear in two sections on the same page. By editing this file, you can add new papers, without having to create any new pages.
+
+### Editing publications
+Open the markdown file at `content/home/publication-cloud.md`. This file should contain information for
+all the papers that are shown on the page. Add a new paper by copying in the following block right below the 
+`publications` line, keeping the indent the same as the files below
+
+    - title: ""
```

Comment:
Can you make this a code block please e.g. use triple back tick on the first and last line to mark this as code in the system:

```
some code
```